### DISCUSSION TITLE
Web link to our sponsors / contributors

### DISCUSSION THREAD

**Discussion by civier** [neurodesk/neurodesk.github.io#137] [Ideas]
Would be nice if the logos on our webpage / Github readme link to the actual institutions/organisations. Hopefully they will also link back to us :-) 

BTW, do you think that linking out will help NeuroDesk to get higher ranking/exposure in Google searches? I don't know enough about it to say.

**Comment by stebo85** [comment]
yes, good idea - go ahead and implement it.
